x/erc20/keeper/ibc_callbacks_integration_test.go

```diff
@@ -240,6 +105,7 @@ var _ = Describe("Convert receiving IBC to Erc20", Ordered, func() {
 
 			s.EvmosChain.SenderAccount.SetSequence(s.EvmosChain.SenderAccount.GetSequence() + 1) //nolint:errcheck
 		})
+		// TODO: Tests are panicking here, need to fix
```

Comment:
is this for now or before we merge the feature to main

Reply:
This was fixed just haven't removed the comment.